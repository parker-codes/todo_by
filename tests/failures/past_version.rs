use todo_by::todo_b4;

fn main() {
    todo_b4!("0.1.0");
    fn my_function() {}
}
