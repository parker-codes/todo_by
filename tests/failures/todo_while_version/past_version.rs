use todo_by::todo_while_version;

fn main() {
    // NOTE: The version is stubbed as 0.3.0 in the test harness.
    todo_while_version!("0.5.0");
    fn my_function() {}
}
