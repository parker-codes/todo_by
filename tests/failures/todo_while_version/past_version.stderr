error: TODO version requirement '^0.5.0' not satisfied by current v0.3.0
 --> tests/failures/todo_while_version/past_version.rs:5:5
  |
5 |     todo_while!("0.5.0");
  |     ^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `todo_while` (in Nightly builds, run with -Z macro-backtrace for more info)
